client/app/styles/tip.css

```diff
@@ -23,3 +24,36 @@
 .fieldAnswer {
   margin-bottom: 10px;
 }
+
+.SubmissionInfoSection {
+  background-color: #eaebe2;
+  border-radius: 2%;
+}
+
+.SubmissionContentSection {
+  background-color: #e5fecd;
+  border-radius: 2%;
+}
+
+.left {
+  text-align: right;
```

Comment:
when one provides special classes for alignment, he should provide the same reversed in rtl.css

e.g. for the .left one should create a .left inside rtl.css with written "text-align: left" instead of "text-align:right"